.env.sample

```
@@ -18,4 +18,9 @@ CORS_MAX_AGE=43200
 ACCESS_DURATION=60
 REFRESH_DURATION=1440
 # Frontend Configuration
-VITE_API_BASE_URL=http://3.239.6.82/api
\ No newline at end of file
+VITE_API_BASE_URL=http://3.239.6.82/api
+
+SHARE_HMAC_SECRET=6c0be1d25c9147b6c9e9f62e8f5c3d27d0f3c4e6a71228d9bb79b05f7c8a4e3f
+APP_BASE_URL=http://localhost:5173
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_BASE_URL key should go before the SHARE_HMAC_SECRET key

(UnorderedKey)